music21/expressions.py

```diff
@@ -558,12 +744,12 @@ def realize(self, srcObj: 'music21.note.Note', *, inPlace=False):
         '''
```

Comment:
update docs to include demonstration of key context, accidental etc.

Reply:
This is now documented in resolveOrnamentalAccidentals header doc.  Is that OK, or should I do it somewhere else?

Reply:
Still working on this.

Reply:
Done.